spec/acceptance/exec_pwsh_spec.rb

```diff
@@ -438,6 +438,18 @@ def platform_string(host, windows, posix)
       it { should be_file }
       its(:content) { should match /puppet/ }
     end
+
+    describe file('/services.txt'), :if => posix6_agents.count > 0 do
```

Comment:
Can we just collapse this into one platform-agnostic test rather than 2 nearly identical tests?

Reply:
Done.